src/content/tools/pub/cmd/pub-token.md

```diff
@@ -42,33 +44,45 @@ Enter secret token: <Type token on stdin>
 ```
 
 :::note
-The token is input on stdin rather than as a command line option to avoid it
-ending up in the shell history such as `~/.bash_history`.
+The token takes input on `stdin` rather than as a command line option
+to keep the token out of the shell history.
 :::
 
-In a scripting situation you can store the secret in an environment variable and
-use `dart pub token add <hosted-url> --env-var <ENV_VAR_NAME>`.
+### Add a credential for all sessions
 
-```console
-$ dart pub token add https://other-package-repo.com/ --env-var TOKEN_VAR
-Requests to "https://other-package-repo.com/" will now be authenticated using the secret token stored in the environment variable "TOKEN_VAR".
-```
+In a script, you can store the secret in an environment variable.
```

Comment:
I think the more critical info for this page/topic should come first in the section, like this:

````suggestion
In a script, you can store the secret token in an environment variable.
Run the following command:

```console
dart pub token add <hosted-url> --env-var <ENV_VAR_NAME>
```

This causes `dart pub get` to read the data stored in `$TOKEN_VAR` then
use it as the authentication token.

```console
$ dart pub token add https://other-package-repo.com/ --env-var TOKEN_VAR
Requests to "https://other-package-repo.com/" will now be authenticated using the secret token stored in the environment variable "TOKEN_VAR".
```
````

Reply:
It [looks like](https://dart-dev--pr5717-fix-3729-y5ryp88s.web.app/tools/pub/cmd/pub-token#add-a-credential-for-all-sessions) you removed all of this content (the syntax/sample code) from the page, accident or intentional?